server/debian/atomiadns-database.postinst

```diff
@@ -597,6 +597,10 @@ if [ $retval != 0 ]; then
 				execute_query "DROP TYPE _algorithmtype"
 			fi
 
+			if [ "$schema_version" -lt 89 ]; then
+				execute_query "INSERT INTO allowed_type (type, synopsis, regexp) VALUES ('ALIAS', 'hostname', '^[a-z0-9_][a-z0-9._-]*$')"
```

Comment:
Add this in ddl.sql scheme file. Otherwise, it won't work after clean installation.